client/src/nv_ingest_client/primitives/tasks/store.py

```diff
@@ -72,17 +89,76 @@ def __str__(self) -> str:
         info += f"  store method: {self._store_method}\n"
         for key, value in self._extra_params.items():
             info += f"  {key}: {value}\n"
+        for key, value in self._params.items():
+            info += f"  {key}: {value}\n"
         return info
 
     def to_dict(self) -> Dict:
         """
         Convert to a dict for submission to redis (fixme)
         """
+
         task_properties = {
             "method": self._store_method,
             "structured": self._structured,
             "images": self._images,
-            "params": self._extra_params,
+            "extra_params": self._extra_params,
+            **self._params,
         }
 
         return {"type": "store", "task_properties": task_properties}
+
+
+class StoreEmbedTask(Task):
```

Comment:
I added this here, because the behavior is most similar to the upload images, where we are writing bytes to a bucket. If it should go somewhere else or as something else, let me know and we can adjust accordingly.